Content.Server/Disease/Effects/DiseaseProgression.cs

```diff
@@ -19,27 +18,9 @@ public sealed class DiseaseProgression : DiseaseEffect
         [ViewVariables(VVAccess.ReadWrite)]
         public float Rate = 0.01f;
```

Comment:
This could use a comment cleanup. "How much to increase the rate by each time this effect happens. 0.01 is 1% progression, 0.1 is 10%, and so on. Use the probability field on the effect to make this more random."